recipes/zmqpp/all/conanfile.py

```diff
@@ -37,24 +41,20 @@ def config_options(self):
 
     def configure(self):
         if self.options.shared:
-            try:
-                del self.options.fPIC
-            except Exception:
-                pass
+            self.options.rm_safe("fPIC")
 
     def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("zeromq/4.3.4")
+        self.requires("zeromq/4.3.4", transitive_headers=True)
```

Comment:
```suggestion
        self.requires("zeromq/4.3.4", transitive_headers=True, transitive_libs=True)
```

https://github.com/zeromq/zmqpp/blob/4.2.0/src/zmqpp/context.hpp#L76-L80 for example

Reply:
@SpaceIm 
Thank you for your detail information.
Now I see.